---
layout: page
title: "Pathogen evolution and population dynamics"
permalink: /research/microbes.html
categories: [project, test]
summary: "How do pathogens spread and evolve?"
---

Pathogens are ideal systems for studying evolution, with their large population sizes, high mutation rates, short generation times, and near-replicate natural experiments in different infections.
They are especially exciting because many of the most fundamental evolutionary questions about the nature of species and genetic diversity are just starting to be answerable for microbes.
At the same time, evolution and population genetics are some of our best lenses for answering practically important questions about pathogen spread and dynamics.

Papers:

- [Shi et al (2025), Genetic drift acts strongly on within-host influenza virus populations during acute infection but does not act alone](https://doi.org/10.1101/2025.08.27.672713)
- [Tran et al (2025), Effects of recombination on multi-drug resistance evolution in _Plasmodium falciparum_ malaria](https://doi.org/10.1371/journal.pcbi.1013401)
- [Raghuram et al (2024), Average Nucleotide Identity based _Staphylococcus aureus_ strain grouping allows identification of strain-specific genes in the pangenome](https://doi.org/10.1128/msystems.00143-24)
- [Mehta et al (2023), Detecting patterns of accessory genome coevolution in _Staphylococcus aureus_ using data from thousands of genomes](https://doi.org/10.1186/s12859-023-05363-4)
- [Koelle et al (2022), 
Masks Do No More Than Prevent Transmission: Theory and Data Undermine the Variolation Hypothesis](https://www.medrxiv.org/content/10.1101/2022.06.28.22277028)
- [Ghafari et al (2022), Investigating the evolutionary origins of the first three SARS-CoV-2 variants of concern](http://dx.doi.org/10.3389/fviro.2022.942555)
- [Scherer et al (2022), SARS-CoV-2 Evolution and Immune Escape in Immunocompromised Patients](https://www.nejm.org/doi/full/10.1056/NEJMc2202861)
- [Allman, Koelle, and Weissman (2022), Heterogeneity in viral populations increases the rate of deleterious mutation accumulation](http://dx.doi.org/10.1093/genetics/iyac127)
- [Ghafari et al (2020), Inferring transmission bottleneck size from viral sequence data using a novel haplotype reconstruction method](https://journals.asm.org/doi/full/10.1128/JVI.00014-20)
- [Li et al (2019), Why are CD8 T cell epitopes of human influenza A virus conserved?](https://journals.asm.org/doi/full/10.1128/JVI.01534-18)
- [Sobel Leonard et al (2017), Transmission Bottleneck Size Estimation from Pathogen Deep-Sequencing Data, with an Application to Human Influenza A Virus](http://jvi.asm.org/content/early/2017/04/27/JVI.00171-17.abstract), and see [correction](https://journals.asm.org/doi/full/10.1128/JVI.00936-19)


